modules/drivers/bigquery-cloud-sdk/test/metabase/test/data/bigquery_cloud_sdk.clj

```diff
@@ -47,15 +47,17 @@
 
 (defn normalize-name
   "Returns a normalized name for a test database or table"
-  ^String [db-or-table identifier]
-  (let [s (str/replace (name identifier) "-" "_")]
-    (case db-or-table
-      ;; All databases created during test runs by this JVM instance get a suffix based on the timestamp from when
-      ;; this namespace was loaded. This dataset will not be deleted after this test run finishes, since there is no
-      ;; reasonable hook to do so (from this test extension namespace), so instead we will rely on each run cleaning
-      ;; up outdated, transient datasets via the `transient-dataset-outdated?` mechanism.
-      :db    (str "v3_" s "__transient_" ns-load-time)
-      :table s)))
+  [identifier]
+  (str/replace (name identifier) "-" "_"))
+
+(defn test-dataset-id
```

Comment:
I made a refactoring, changing `(normalize-name :db database-name)` to `(test-dataset-id database-name)`. 

The reason behind this change is there was a bug [here](https://github.com/metabase/metabase/pull/35964/files#diff-02e0ddde9d998d54b97680780db70f4ee393bcb76c6a4ecce56c9693ca947236R345) where a raw database-name was passed to `destroy-dataset!`. I believe that bug was caused by confusion around `database-name`, `dataset-name`, and `dataset-id`, all being used to refer to the same type of thing. With this refactoring I wanted to make it clearer that `destroy-dataset!` should always take a test `dataset-id`, which is distinct from `database-name`, which is a key on the db definition.